Dockerfile

```diff
@@ -1,13 +1,99 @@
-ARG DOCKER_ORG=usdotfhwastoldev
-ARG DOCKER_TAG=develop
-FROM ${DOCKER_ORG}/carma-base:${DOCKER_TAG} as base_image
-SHELL ["/bin/bash", "-c"]
+FROM ros:humble-ros-base-jammy as base-install
 
 ARG DEBIAN_FRONTEND="noninteractive"
+ARG GIT_BRANCH=develop
+
+# Make sure bash catches errors (no need to chain commands with &&, use ; instead)
+SHELL ["/bin/bash", "-o", "pipefail", "-o", "errexit", "-c"]
+
+###########################
+# 1.) Bring system up to the latest ROS desktop configuration
+###########################
+RUN apt-get -y update
```

Comment:
Can we combine the multiple RUN statements here ?

Reply:
we are going to skip this as we discussed